Docs/Profiling.md

```diff
@@ -0,0 +1,80 @@
+
+![Profiling](Images/FeatureCards/Profiling.png)
+
+## Visual profiler
+
+To continuously track performance over the lifetime of development, it is highly recommended to always show a framerate visual while running & debugging an application. Graphics Tool's provides the `GTVisualProfiler` actor which gives real-time information about the current frame times represented in milliseconds, draw call count, and visible polygon count in a stereo friendly application view.
+
+> [!NOTE]
+> To utilize the visual profiler simply drop an instance of the `GTVisualProfiler` actor in a level. To disable the profiler you can mark it as hidden in game (or not visible).
+
+The visual profiler provides four metrics around frame performance and two metrics around level complexity as outlined below.
+
+| Name       | Description                                                                                                                                                                                                                                 |
+|------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Frame      | Frame time represents the total amount of time spent generating one frame of the app. Since both the Game and Draw threads sync up before finishing a frame, frame time is often close to the time being shown in one of these threads. |
+| Game       | If Frame time is close to Game time, the app's performance is likely being bottlenecked (negatively impacted) by the game thread. The game thread is resposible for most app logic, especially Blueprint logic.                             |
+| Draw       | If Frame time is close to Draw time, the game's performance is likely being bottlenecked by the rendering thread. The rendering thread is responsible for decideding what to render and submitting work to the GPU.                         |
+| GPU        | GPU time measures how long the video card took to render the scene. Since GPU time is synced to the frame, it will likely be similar to Frame time.                                                                                        |
+| Draw Calls | Draw calls can be thought of as the number on times a graphics API (such as [DirectX](https://en.wikipedia.org/wiki/DirectX)) is told to render an object.                                                                                  |
```

Comment:
> Draw Calls [](start = 1, length = 11)

I'm playing with the Profiling map in the editor and I don't see the number of draw calls change as I change the number of Landers. Is that expected?

Reply:
I'm glad you brought this up since it is "expected" but a bit obscure as to why! In a recent release Epic enable "auto instancing" which will automatically batch draw calls together and submit them as an instanced draw call. Unfortunately, this is not enabled on mobile by default. So you have to do it manually, which I have [commented out in a project config](https://github.com/microsoft/MixedReality-GraphicsTools-Unreal/blob/bb655ba5a7105178ca3c187700082fe79f490fe7/GraphicsToolsProject/Config/DefaultEngine.ini#L121). 

So, in PIE you should not see a change on the lander draw calls but you will on HoloLens without changing that config.